index.js

```diff
@@ -38,23 +39,42 @@ module.exports = class SeedBee extends ReadyResource {
 
   async get (key, opts) {
     key = Id.decode(key)
-    const entry = await this.bee.get(key, opts)
+    const entry = await this.bee.get(key, { ...opts, ...contentEncoding })
     return entry ? entry.value : null
   }
 
   async del (key) {
     key = Id.decode(key)
-    return this.bee.del(key) // TODO: cas?
+    return this.bee.del(key, contentEncoding) // TODO: cas?
   }
 
   async * entries (opts = {}) {
-    for await (const e of this.bee.createReadStream()) {
+    for await (const e of this.bee.createReadStream({ ...opts, ...contentEncoding })) {
```

Comment:
This should not pass `opts`, just `createReadStream(contentEncoding)` to keep it like before